<launch>
  <node name="docker_master" pkg="rosdop" type="docker_master.py" output="screen">
    <param name="use_dnsmasq"         type="bool"   value="True"/>
    <param name="dnsmasq_node_name"   type="String" value="my_dnsmasq"/>
  </node>
</launch>
